package.json

```diff
@@ -4,6 +4,9 @@
     "description": "Opinionated logging for Node projects",
     "main": "lib",
     "repository": "https://github.com/globality-corp/nodule-logging",
+    "jest": {
```

Comment:
I remember seeing this error in other projects when we updated some library versions. Not sure why it appeared now expect maybe some libraries were way out of date.